### DISCUSSION TITLE
无法授权其他设备

### DISCUSSION THREAD

**Discussion by Capel-Allen** [getlantern/lantern-forum-cn#423] [Q&A]
找不到授权其他设备，我现在只能在一台设备登录了

**Comment by MaxMKMax** [comment]
@Capel-Allen 您授权的方法应该是个别设备去授权的. 可以参考这里:
https://lantern.io/zh-Hans/faq

如果您下载了最新版本就应该没这个问题了. 您试试看好吗? 谢谢哦! 

我们这周发布了新的 Flutter 桌面应用程序测试版（8.0.0 版本）。

Mac - https://github.com/getlantern/lantern-binaries/raw/main/lantern-installer-preview.dmg

Windows - https://github.com/getlantern/lantern-binaries/raw/main/lantern-installer-preview.exe

Linux - https://github.com/getlantern/lantern-binaries/raw/main/lantern-installer-preview-64-bit.deb
